client/src/scss/section.scss

```diff
@@ -0,0 +1,109 @@
+@import "./fonts";
+@import "./layout";
+@import "./media-queries";
+@import "./mixins";
+@import "./variables";
+@import "./themes";
+
+.section {
+  display: flex;
+  flex-direction: column;
+  padding: 66px 0 24px 0;
+  border-bottom: 1px solid var(--border-color);
+
+  @include tablet-down {
+    padding-top: 24px;
+  }
+
+  .section--header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 34px;
+  }
+
+  .section--data {
+    margin-bottom: 26px;
+
+    .label {
+      @include font-size(12px);
+
+      margin-bottom: 4px;
+      color: $gray-6;
```

Comment:
Change color for dark theme from $gray-6 to $gray-5.

Reply:
Resolved